### PR TITLE
Chapter 3 Static Pages

### PR DESCRIPTION
**Pull request by aisharina99**

### CODE REVIEW

**Comment by aisharina99** [comment]
ready

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -13,3 +13,7 @@
  *= require_tree .
  *= require_self
  */
+
+
```
Comment (ghost) [inline app/assets/stylesheets/application.scss]: delete blank line

File: app/views/static_pages/help.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<%= provide :title, "Help" %>
+
+<h1>Help</h1>
+<p>
+  Get help on the Ruby on Rails Tutorial at the
+  <%= link_to "Rails Tutorial help page", "http://www.railstutorial.org/help" %>.
+  To get help on this sample app, see the
+  <%= link_to "Ruby on Rails Tutorial Book", "http://www.railstutorial.org/book" %>
+
```
Comment (ghost) [inline app/views/static_pages/help.html.erb]: delete blank line

**Review by ghost — COMMENTED** [review]
cmt

**Comment by aisharina99** [comment]
ready

File: app/views/static_pages/help.html.erb (hunk 2 above)
Comment (aisharina99) [reply app/views/static_pages/help.html.erb]: sorry i change follow my friend

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
   </head>
 
   <body>
+  <%= render "layouts/header" %>
+  <div class="container">
     <%= yield %>
-  </body>
+  </div>
+
+</body>
```
Comment (ghost) [inline app/views/layouts/application.html.erb]: ident please

File: app/helpers/static_pages_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module StaticPagesHelper
```
Comment (ghost) [inline app/helpers/static_pages_helper.rb]: if you don't use this file, please delete it

**Review by ghost — COMMENTED** [review]
cmt

**Comment by aisharina99** [comment]
ready

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -1,16 +1,20 @@
...
 
-    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
-    <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= stylesheet_link_tag "application", media: "all", "data-turbolinks-track": "reload" %>
+    <%= javascript_pack_tag "application", "data-turbolinks-track": "reload" %>
   </head>
 
   <body>
+    <%= render "layouts/header" %>
+  <div class="container">
```
Comment (huongnt-2545) [inline app/views/layouts/application.html.erb]: line 15 -> 17
```suggestion
    <div class="container">
      <%= yield %>
    </div>
```

File: test/fixtures/users.yml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (huongnt-2545) [inline test/fixtures/users.yml]: delete this file, it's no need

File: test/models/user_test.rb [hunk 7]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (huongnt-2545) [inline test/models/user_test.rb]: delete this file, it's no need

**Comment by aisharina99** [comment]
ready

File: app/views/layouts/application.html.erb (hunk 5 above)
Comment (ghost) [reply app/views/layouts/application.html.erb]: please indent your code

**Review by ghost — COMMENTED** [review]
cmt

File: app/views/layouts/application.html.erb (hunk 5 above)
Comment (aisharina99) [reply app/views/layouts/application.html.erb]: is it okay now?

**Comment by aisharina99** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**